src/AppBundle/Controller/BlurayController.php

```diff
@@ -14,8 +13,22 @@ class BlurayController extends Controller
      */
     public function listingAction($page = 1, $maxperpage = 4)
     {
-        $listBluray = $this->getDoctrine()->getRepository('AppBundle:Bluray')->findAll();
-        return ['listBluray' => $listBluray];
+        $blurayRepository = $this->getDoctrine()->getRepository('AppBundle:Bluray');
+        $query = $blurayRepository->createQueryBuilder('b')
+            ->setFirstResult(($page-1) * $maxperpage)
+            ->setMaxResults($maxperpage)
+            ->getQuery()
+        ;
```

Comment:
Put your query in a dedicated repository (eg. AppBundle\Repository\BlurayRepository, AppBundle\Entity\BlurayRepository or AppBundle\Entity\Repository\BlurayRepository)

See https://github.com/Jean-Beru/dunkerque/tree/master/src/AppBundle/Entity.